src/Module/Contacts.php

```diff
@@ -0,0 +1,1137 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\App;
+use Friendica\BaseModule;
+use Friendica\Content\ContactSelector;
+use Friendica\Content\Nav;
+use Friendica\Content\Text\BBCode;
+use Friendica\Content\Widget;
+use Friendica\Core\Addon;
+use Friendica\Core\L10n;
+use Friendica\Core\Protocol;
+use Friendica\Core\System;
+use Friendica\Core\Worker;
+use Friendica\Database\DBA;
+use Friendica\Model\Contact;
+use Friendica\Model\GContact;
+use Friendica\Model\Group;
+use Friendica\Model\Profile;
+use Friendica\Network\Probe;
+use Friendica\Util\DateTimeFormat;
+use Friendica\Util\Proxy as ProxyUtils;
+use Friendica\Core\ACL;
+use Friendica\Module\Login;
+
+/**
+ *  Manages and show Contacts and their content
+ *
+ *  @brief manages contacts
+ */
+class Contacts extends BaseModule 
```

Comment:
I know it's an endeavor, but can you please rename this module to just `Contact` and chase down the project anywhere `contacts` is used in a URI string and replace it with `contact`?